mParticle-Apple-SDK/Utils/MPDevice.swift

```diff
@@ -217,7 +217,8 @@ import CoreTelephony
                 if let vendor = userDefaults[Device.kMPDeviceAppVendorIdKey] as? String, vendor != Device.kMPDeviceInvalidVendorId {
                     _vendorId = vendor
                 } else {
-                    _vendorId = UUID().uuidString
+                    _vendorId = UIDevice.current.identifierForVendor?.uuidString
+                    userDefaults[Device.kMPDeviceAppVendorIdKey] = _vendorId
```

Comment:
Heres the [old functionality](https://github.com/mParticle/mparticle-apple-sdk/commit/0fda3479b85db4ad5f09a0b193f14162944d140e#diff-2f76e6505603ebc6dcc124be5432a7b9f9f43838724a814e48e395ebff7f3866L297) to compare this more easily. I think this is the correct fix comparing these